install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -6,9 +6,11 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
     app: {{ template "istio.name" . }}
+    {{- if .Values.global.enableTillerLabels }}
```

Comment:
tiller injects these labels on creation of the object.  The labels are also immutable.  WHy